lib/tasks/invite_code.rake

```diff
@@ -1,16 +1,21 @@
 
 
 desc "Generate a batch of invite codes"
-task :generate_invites, [:n] => :environment do |task, args|
+task :generate_invites, [:n, :u] => :environment do |task, args|
 
-    codes = args.n
-    codes = codes.to_i
+  codes = args.n
+  codes = codes.to_i
 
-    codes.times do |x|
+  if args.u != nil
+    max_uses = args.u
+  end
+
+  codes.times do |x|
     x = InviteCode.new
+    x.max_uses = max_uses
```

Comment:
does this work if max_users is not defined - i.e. if you don't supply an argument for u?